packages/sitecore-jss-rendering-host/package.json

```diff
@@ -45,6 +46,7 @@
     "@types/webpack-dev-middleware": "2.0.5",
     "@types/webpack-dev-server": "3.1.7",
     "chai": "^4.3.0",
+    "compression": "^1.7.2",
```

Comment:
I'm curious, but why _compression_ is added to _dependencies_ and _devDependencies_? It should be enough to just include it in _dependencies_

Reply:
Yup, that's an oversight on my part.